.golangci.yml

```diff
@@ -1,5 +1,7 @@
 run:
+  concurrency: 8
```

Comment:
This one probably should be 4, because default Ci node has 4 CPU

Reply:
Yes, it's a temporary change to just understand what is happening in CI. The lint stage keeps timing out in CI because the linter is taking ~8 minutes to run whereas it only takes 17 seconds on my laptop. :confused:

Reply:
@charith-elastic Does this repro on your laptop in the docker container? `make -C build/ci ci-interactive` from repo root and then try running linter.